<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<link type="text/css" rel="stylesheet" href="css/experience.css"  media="screen,projection"/>
<style>
</style>
</head>
<body>
<a href="index.html"><img src="image/logo_cut.png" class="logo" width="176" height="110" alt="Logo Cut"></a>
  <p>&nbsp;</p>  <p>&nbsp;</p>    <p>&nbsp;</p>  
  <div class='card-holder'>
  <div class='card-wrapper'>
    <a href='#'>
      <div class='card bg-01'>
        <a href="index.html"><span class='card-content'>HOME</span></a>
      </div>
    </a>
  </div>
  <div class='card-wrapper'>
    <a href='#'>
      <div class='card bg-02'>
         <a href="about.html"><span class='card-content'>ABOUT ME</span></a>
      </div>
    </a>
  </div>
  <div class='card-wrapper'>
    <a href='#'>
      <div class='card bg-03'>
         <a href="Gallery/gallery.html"><span class='card-content'>WORK COLLECTION</span></a>
      </div>
    </a>
  </div>
  <div class='card-wrapper'>
    <a href='#'>
      <div class='card bg-05'>
        <a href="graphic.html"><span class='card-content'>GRAPHIC DESIGN</span></a>
      </div>
    </a>
  </div>
  <div class='card-wrapper'>
    <a href='#'>
      <div class='card bg-04'>
         <a href="software.html"><span class='card-content'>SOFTWARE ENGINEER	</span></a>
      </div>
    </a>
  </div>
  <div class='card-wrapper'>
    <a href='#'>
      <div class='card bg-06'>
        <a href="contact.html"><span class='card-content'>CONTACT ME</span></a>
      </div>
    </a>
  </div>
</div>

<div class="right">
	<div class="container">
		<h1 style="color:#6d9c1b" class="center">Selected Projects</h1>
  		<a name="img"></a>
  		<h3>Mutation of Antibiotic Resistance Gene analysis in <i>Enterococcus faecalis</i> </h3>
  		<ul>
  			<li>May, 2015 ~ Aug, 2015</li>
  			<li>Philips Research, Briarcliff Manor, NY, USA</li>
  			<li><strong>ABSTRACT:</strong> 
  			<p>Antibiotic resistance bacteria are consistently causing global health crisis and are difficult to treat. This project aims to discover putative antibiotic resistance related genes in Enterococcus faecalis samples collected from clinical patients in Westchester Medical Center by nasal swab. Part of these samples show resistant phenotype against Daptomycin treatment while the remaining samples remain sensitive. 
The DNA sequences of all these samples are obtained using next generation sequencing. Next these sequences are filtered by variant calling and the information of SNP (single nucleotide polymorphism)  position is obtained. 
Next all these sequences are annotated by a bioinformatics tool called Prokka to predicted the putative gene function.
I also employed statistical method to verify the result. I performed Chi-square test over the samples using R to see how significant the SNP allele is related to resistance and sensitive samples respectively. P values for each SNP are calculated to show the correlation.
I developed a program in Python to automatically determine whether a SNP is inside a gene coding region in chromosome and therefore causing a mutation. If so, it would predict the animo acid coded by the corresponding codon and classify the mutation into synonymous(causing functional change in protein product) or non-synonymous(the mutation lead to the same animo acid and no change in the protein product). </p>
		</ul>
  		<hr/>
  		<h3>Bioimage Sequences Processing and Biological Image Data Mining</h3>
  		<ul>
  			<li>Jan, 2015 ~ Present</li>
  			<li>Carnegie Mellon University, PA, USA</li>
  			<li><strong>ABSTRACT:</strong> 
  			<p>Analyzed and classified the noise distribution of background noises cropped from a series of fluorescence images of <i>Drosophila</i> vesicle transport. An example image is shown as Fig.1. </p>
  		</ul>
  		  <img src="project/03.jpg" width="550" height="100" class="displayed">
  		  <p class="center"><strong>Fig 1.</strong> An example fluorescence image of <i>Drosophila</i> vesicle transport.</p>
  	  </br>
		  <img src="project/space.jpg" width="500" height="450" class="displayed">
		  <p class="center"><strong>Fig 2.</strong> Histogram of background noise distribution of Fig.1 changed over space.</p>
	  </br>
  		  <img src="project/w3.jpg" width="500" height="450" class="displayed">
  		  <p class="center"><strong>Fig 3.</strong> Power spectrum graphs for noise analysis of Fig.1 (The result indicates a white noise feature).</p>
  	  </br>
  		  <img src="project/4.jpg" width="300" height="250" class="displayed">
  		  <p class="center"><strong>Fig 4.</strong> Orignial bioimage data of yeast (S. Matsumoto, M. Hayano, Y. Kanoh, H. Masai, Multiple pathways can bypass the essential role of fission yeast Hsk1 kinase in DNA replication initiation, Journal of Cell Biology, 195:387-401, 2011.).</p>
	  </br>	  
  		  <img src="project/w1.jpg" width="500" height="450" class="displayed">
  		  <p class="center"><strong>Fig 5.</strong> Power spectrum graphs for noise analysis of Fig.4.</p>
  	  </br>
  		  <img src="project/5.jpg" width="500" height="450" class="displayed">
  		  <p class="center"><strong>Fig 6.</strong> Orignial bioimage data of a synapse (Baez, L. Luchelli, D. Maschi, M. Habif, M. Pascual, M. Thomas, G. Boccaccio, Smaug1 mRNA-silencing foci respond to NMDA and modulate synapse formation, Journal of Cell Biology 195:1141-1157, 2011.).</p>
  		  <img src="project/w2.jpg" width="400" height="350" class="displayed">
		  <p class="center"><strong>Fig 7.</strong> Power spectrum graphs for noise analysis of Fig.6</p>
		
		
		<a name="perl"></a>
		<hr/>
		<h3>Perl-based Analysis of CRISPR/Cas9-induced DNA Sequences Differences</h3>
		<ul>
			<li>Mar, 2014 ~ Jun, 2014</li>
			<li>Sun Yat-sen University, Guangzhou, China</li>
			<li><strong>ABSTRACT:</strong> 
			<p>The typeII CRISPR/Cas9 system is an effective way of genome engineering to facilitate RNA-guided site-specific genomes edit by designing sgRNA to target DNA cleavage. The DNA double-strand breaks can stimulate homology- directed repair and induce minimal NHEJ (Non-Homologous End Joining)-mediated indel (insertion/deletion) mutation. However, it is still a conundrum to precisely predict the exact sequence changes in the targeted loci. In order to analyze sequence variations in genome editing using CRISPR/Cas9 system on a base-level, I developed a Perl program to analyze sequence data and calculate the percentages of insertion/deletion(indel) and the mutation types(frame-shift/in-frame) based on the differences output generated by Codon Code Aligner software after aligning sequence samples. The program can also calculate the amounts of samples according to different indel sizes. The sample sequences used in this study are the sequencing results of EMX1 gene and B2M gene from human 293T cells and CD274 gene and B2M gene from human H441 cells after being edited by CRISPR/Cas9 system. The results show that all the sequence has a high percentage of both indel and frame-shift mutation. Most of the indel pattern are deletion. The results indicate that CRISPR/Cas9 System is efficient in causing loss of function in encoded proteins. It also indicates that it is practical to analyze indel statistic using the Perl program cited in this study.</p>
		</ul>
		  <img src="project/CD274 alignment.jpg" width="800" height="750" class="displayed">
		  <p class="center"><strong>Fig 1.</strong> H441-CD274 sample sequences alignment using CodonCode Aligner software (indel region)</p>
	  </br>
		  <img src="project/EMX1 alignment.jpg" width="800" height="750" class="displayed">
		  <p class="center"><strong>Fig 2.</strong> 293T-EMX1 sample sequences alignment using CodonCode Aligner (indel region)</p>
		  <img src="project/emx1.png" width="400" height="350" class="displayed">
		  <p class="center"><strong>Fig 3.</strong> 293T-EMX1 sample sequences alignment using CodonCode Aligner (indel region)</p>
		  <img src="project/CD274.png" width="400" height="350" class="displayed">
		  <p class="center"><strong>Fig 4.</strong> 293T-EMX1 sample sequences alignment using CodonCode Aligner (indel region)</p>
		  <img src="project/b2m.png" width="400" height="350" class="displayed">
		  <p class="center"><strong>Fig 5.</strong> 293T-EMX1 sample sequences alignment using CodonCode Aligner (indel region)</p>
		  <img src="project/h441-b2m.png" width="400" height="350" class="displayed">
		  <p class="center"><strong>Fig 6.</strong> 293T-EMX1 sample sequences alignment using CodonCode Aligner (indel region)</p>
		  <p>&nbsp;</p>
		  <p>&nbsp;</p>
  		<hr/>
  		<a name="net"></a>
  		<p>&nbsp;</p>
  		<h3>Visualization of the interaction networks of small non-coding RNAs and RNA-Binding Proteins</h3>
		<ul>
  		  <li>Carnegie Mellon University, PA, USA, 2014 Fall</li>
		  <li>Analyze the 562 interactions of 329 sncRNAs with each of the 5 RNA-Binding Proteins and built two networks using R.</li>
  		  <li>Designed a poster for my course project of Introduction to Network Science.</li>
	  </ul>
	  <strong>ABSTRACT:</strong> 
		  <p>Protein-RNA interaction is a very important issue in biological study. The data I used in this project are all exacted from a database call starBase(<a href = "http://starbase.sysu.edu.cn/" style="color:#75a8e7">http://starbase.sysu.edu.cn/</a>), which focuses on decoding Pan-Cancer and Interaction Networks of small non-coding RNA and RNA-binding proteins (RBPs) from large-scale CLIP-Seq (HITS-CLIP, PAR-CLIP, iCLIP, CLASH) data. And here visualizing the interaction network can give us a more clear view on the relationship between small-noncoding  RNA and RNA-binding protein. The results show that some RBPs interact with most of the non-coding RNA, which might indicate that these proteins might play important roles in cancer pathway and therefore shed light for researcher to develop new clinical treatments.</p>
  		  <img src="image/projectimg/Rposter.jpg" width="800" height="1200" alt="Rposter" class="displayed">
		  <a name="igem"></a>
		<hr/>
		<h3>2013 iGEM SYSU-Software team member (Main contributor)</h3>
		<ul>
		  <li>Boston, USA, 2013 Fall</li>
		  <li>Designed User Experience interface for the award-winning Software CAST-Designer.</li>
		  <li>Visualized the idea and made illustractions for the project.</li>
		  <li>Designed a detailed poster explaining our project.</li>
		  <img src="igem/poster_world2s copy.jpg" width="800" height="800" alt="poster" class="displayed">
		  <li><strong>HONORS</strong></li>
	  </ul>
		  <p>  - Received Best software project in 2013 iGEM (International Genetic Engineering Machine) world competition</p>
		  <P>  - Received Gold Award in 2013 iGEM (International Genetic Engineering Machine) world competition</P>
		  <a href="http://2014.igem.org/Tracks/Software" class="wordlink"><img src="image/software track.png" width="800" height="450" alt="Software Track" class="displayed"></a>
		  <p class="left">A screen shot of the software track of 2013 iGEM official site. The illustration was made by me^_^ ! Click the picture to see the original website.</p>
		  <p>&nbsp;</p>
		  <p>&nbsp;</p>
	</div>
	
		    <p class="bottom">
	    This website is designed and developed by Xinran Li.© Copyright reserved.</p>
	</div>

</body>
</html>
