<html>
  <head>
    <link type="text/css" rel="stylesheet" href="css/materialize.min.css"  media="screen,projection"/>
	<link type="text/css" rel="stylesheet" href="css/compbio.css"  media="screen,projection"/>
	<link rel="shortcut icon" src="images/icon/favicon.gif">
  </head>
  <body>
    <!--Import jQuery before materialize.js-->
    <script type="text/javascript" src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
    <script type="text/javascript" src="js/materialize.min.js"></script>

    <style>
	</style>
	<img src="images/sci.png" width="180" height="460" alt="Sci" class="align:left" style="position: absolute; bottom: 0px;left: 10px;">
	<div class="container; left" >
	<h1 style="position: absolute; left: 40px;">Bioinformatics & Computational Biology!</h1></div>
	<div class="scroll""right""align:left">
		<h5><strong>Hello! I am a master student in Computational Biology at Carnegie Mellon University!</strong></h5>
		<p style= "position: absolute; bottom: 0px; left: 10px;">
		<p class="color:#502b8c"><strong>Biological experiments</strong>: Animal and Plant Anatomy; Cell and Nuclear Staining; Human Cell Culture; Plant Tissue Culture; Bacterial Culture; Accurate Cell Counting; Antibody Testing (ELISA); Electrophoresis (Agarose gel and SDS-PAGE); Western Blot; DNA/RNA/Plasmid Extraction; Micro Injection; Immunohistochemistry; Gene Cloning; TA Cloning; Quantative-PCR; etc.</p>
		<p class="color:#502b8c"><strong>Programming Skills:</strong> Go, Python, Perl, R, PASCAL, LaTeX, HTML5, CSS3, Javascript, Matlab</p>
<p class="color:#502b8c"><strong>Selected Coursework:</strong></p>
<p>Machine Learning; Bioimage informatics; Web Application Development; Programming for Scientists; Introduction to Network Science; Higher mathematics; Physiology; Biostatistics; Bioinformatics; Applied Cell and Molecular Biology; Cell Biology; Molecular Biology; Genetics; Oncology; Mircrobiology.</p> 
<p></p>
		</h1>
		<p></p>
		<p></p>
		<p class="text-align:left;">Projects:</>
		<ul>
		<li>
		<p><strong>Perl-based Analysis of CRISPR/Cas9-induced DNA Sequences Differences</strong></p>
		<p><strong>Abstract:</strong>Wrote a Perl program to analyze sequence data and calculate the percentages of insertion/deletion and the mutation types (frame-shift/in-frame) based on the differences output generated by Codon Code Aligner software after aligning sequence samples. The program can also calculate the amounts of samples according to different indel sizes. <br/> 
			Samples: EMX1 gene and B2M gene from human 293T cells and CD274 gene and B2M gene from human H441 cells after being edited by CRISPR/Cas9 system. <br/>
			Results: all the sequence has a high percentage of both indel and frame-shift mutation. Most of the indel pattern are deletion. The results indicate that CRISPR/Cas9 System is efficient in causing loss of function in encoded proteins. It also indicates that it is practical to analyze indel statistic using the Perl program cited in this study.</p>
		<a href=""><p>Click here to view details.</p></a></li>
		<li>
		<p><strong>Processed bioimage sequences and biological image data mining</strong></p>
		<a href="compbioexp.html"><p>Click here to view details.</p></a></li>
		<li>
		<p></p>
		</li>
	</ul>
	</p></div>
<br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/>
	<p style="text-align:center; position: absolute; bottom: 30px; left: 50%;"><a href="index.html">>>BACK<<</a></p>
	<p style="text-align:center; position: absolute; bottom: 0px; left: 35%">This website is designed and developed by Xinran Li&copy; All right reserved.</p>
</body>
</hteml>