<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<style>
div.footer {
    background-color: #70a19f;
	text-align: center;
	font-size: 3px;
	color: #70a19f;
}

.right {
    position: absolute;
    right: 350px;
    width: 500px;
    background-color: #FFFFFF;
	opacity:0.8;
}
.left{
    position: absolute;
    left: 130px;
}

.bottom{
	left: 10px;
	bottom: 10px;
	font-size: 13px;
	color: #70a19f;
}

a:link,
a:hover,
a:visited,
a:active {
  color: #fff;
  text-decoration: none;
}
a.wordlink{
  color:#006a80;
  text-decoration: none;
}

body {
  height: 100%;
  width: 100%;
  margin: 0;
  padding: 0;
  
  background: #fff;
}

.card-holder {  
  position: fixed;
  width: 0px;
  overflow: visible;
}

.card-wrapper {
  display: inline-block;
  float: right;
  clear: both;
}

.card {
  position: relative;
  left: 32px;
  padding: 16px 32px 16px 64px;
  margin: 8px;  
  x-box-shadow: 0 0 8px 0px rgba(0, 0, 0, 0.5);
  box-shadow: 8px 0 8px -8px rgba(0, 0, 0, 0.5);
  background: #fff;
  transition: all 0.3s ease-in-out 0.1s;
}

.card:hover {
  position: relative;
  left: 100%;
  margin-left: -32px;
  box-shadow: 
    0 -8px 8px -8px rgba(0, 0, 0, 0.5),
    0 8px 8px -8px rgba(0, 0, 0, 0.5);
  transition: all 0.3s ease-in-out;
}

.card-content {
  display: inline-block;
  color: #fff;
  font-family: 'Droid Sans', sans-serif;
  font-size: 16px;
  font-weight: bold;
  white-space: nowrap;
}

.bg-01 { background: #93D8FA; }
.bg-02 { background: #B4D9ED; }
.bg-03 { background: #DADAED; }
.bg-04 { background: #FFD9D9; }
.bg-05 { background: #EEE8AB; }
.bg-06 { background: #F4F293; }

.logo{
	position: fixed;
}
</style>
</head>
<body>
<a href="index.html"><img src="image/logo_cut.png" class="logo" width="176" height="110" alt="Logo Cut"></a>
<div class="right"><br/><br/><br/><img src="image/project logo.png" width="193" height="61" alt="Project Logo"></div>
  <p>&nbsp;</p>  <p>&nbsp;</p>    <p>&nbsp;</p>  
  <div class='card-holder'>
  <div class='card-wrapper'>
    <a href='#'>
      <div class='card bg-01'>
        <a href="home.html"><span class='card-content'>HOME</span></a>
      </div>
    </a>
  </div>
  <div class='card-wrapper'>
    <a href='#'>
      <div class='card bg-02'>
        <span class='card-content'>ABOUT ME</span>
      </div>
    </a>
  </div>
  <div class='card-wrapper'>
    <a href='#'>
      <div class='card bg-03'>
        <span class='card-content'>WORK COLLECTION</span>
      </div>
    </a>
  </div>
  <div class='card-wrapper'>
    <a href='#'>
      <div class='card bg-04'>
        <span class='card-content'>GELLERY</span>
      </div>
    </a>
  </div>
  <div class='card-wrapper'>
    <a href='#'>
      <div class='card bg-05'>
        <span class='card-content'>PORFOLIO</span>
      </div>
    </a>
  </div>
  <div class='card-wrapper'>
    <a href='#'>
      <div class='card bg-06'>
        <span class='card-content'>CONTACT ME</span>
      </div>
    </a>
  </div>
</div>

<div class="right">
<h3>Interaction network of small non-coding RNA and RNA-binding Proteins</h3>
  <p>Course project for Introduction to network science</p>
  <p>WHEN: 2014 Fall</p>
  <p>WHERE: CMU, USA</p>
  <p>Used R to visualize the interaction network of small non-coding RNA and RNA-binding Proteins interaction based on the data extracted from starBase.</p>
  <img src="projects/Rplot01.png" width="500" height="410" alt="R">
  <img src="projects/poster.jpg" width="500" height="610" alt="poster">
  <p >Poster project of Interaction network of small non-coding RNA and RNA-binding Proteins</p>
 
 
  <p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p>
  <p>Undergraduate thesis research</p>
  <h3>Analyzed base-pair variation pattern in CRISPR-edited DNA sequence</h3>
  <p> Analyzed the alignment results of sequences edited by type II CRISPR/Cas9 system from 120 samples from EMX1 and B2M of 293T cell line and CD274 and B2M of H441 cell line using CodonCodeAligner.</p>
<p>Used Perl to calculate the frequency of insertion/deletion and frameshift/inframe mutation based on the analysis above.</p>
<p>[Abstract] The type II CRISPR/Cas9 system is an effective way of genome engineering to facilitate RNA-guided site-specific genomes edit by designing sgRNA to target DNA cleavage. The DNA double-strand breaks can stimulate homology- directed repair and induce minimal NHEJ (Non-Homologous End Joining)-mediated indel (insertion/deletion) mutation. However, it is still a conundrum to precisely predict the exact sequence changes in the targeted loci. In order to analyze sequence variations in genome editing using CRISPR/Cas9 system on a base-level, this study has developed a Perl program to analyze sequence data and calculate the percentages of insertion/deletion and the mutation types (frame-shift/in-frame) based on the differences output generated by Codon Code Aligner software after aligning sequence samples. The program can also calculate the amounts of samples according to different indel sizes. The sample sequences used in this study are the sequencing results of EMX1 gene and B2M gene from human 293T cells and CD274 gene and B2M gene from human H441 cells after being edited by CRISPR/Cas9 system. The results show that all the sequence has a high percentage of both indel and frame-shift mutation. Most of the indel pattern are deletion. The results indicate that CRISPR/Cas9 System is efficient in causing loss of function in encoded proteins. It also indicates that it is practical to analyze indel statistic using the Perl program cited in this study.</p>
  <img src="projects/fig1.jpg" width="800" height="410" alt="FIG1">
  <p>293T-EMX1 sample sequences alignment using CodonCode Aligner (indel region)</p>
  <img src="projects/f2.jpg" width="800" height="410" alt="FIG2">
    <p>H441-CD274 sample sequences alignment using CodonCode Aligner (indel region)</p>
  <img src="projects/f3.jpg" width="300" height="410" alt="FIG3">
    <p>Analysis of 293T-EMX1Samples using Perl script </p>
  
  <p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p>
  <h3>Designed User Experience and interface for Software CAST-Designer </h3>
  <p> Team Project for SYSU-Software in 2013 iGEM competition.</p>
  <p> CAST-Designer stands for Computer Aided Synbio Tool, which is an integrated tool for Synthetic Biology.</p>
<p>Used Perl to calculate the frequency of insertion/deletion and frameshift/inframe mutation based on the analysis above.<br /> I designed the posters and propaganda materials for our team.</p>
  <p><a href="http://2013.igem.org/Team:SYSU-Software" class="wordlink">Chick Here to visit team website!</a></p>
  <p class="bottom">
  This website is designed and developed by Xinran Li.Â© Copyright reserved.</p>
</div>

</body>
</html>
